src/main/java/com/dev/eficiente/nosso/bolao/NossoBolaoApplication.java

```diff
@@ -7,6 +9,8 @@
 public class NossoBolaoApplication {
 
 	public static void main(String[] args) {
+		TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
```

Comment:
Aqui é para o timezone da app fica igual ao do banco? Pq se for, eu acho que você pode ter um problema. Quando registrarmos um instante, por exemplo 11 da manhã, vai rolar a aplicação do fuso para utc. Quando na verdade você queria 11 da manhã do brasil e provavelmente 15 utc.

Reply:
Aqui seria para padronizar como UTC mesmo.
Então o instante salvo as 11 está indo para o banco 14h. 
Se não deixar com um padrão também terá problemas de dados salvos em fuso diferente, ceto?

Reply:
Entendi e me precipitei no comentário. Fiquei enviesado em relação a como eu trato. O problema de definir como UTC é que você vai ter que trabalhar em cima do timezone todo o tempo que for exibir essa data. Se você definir o timezone do sistema e o usado para guardar as datas para o país onde o sistema pretende rodar, Brasil por exemplo, você evita isso. 

Geralmente eu configuro na string de conexão do banco e já subo a aplicação passando o argumento da jvm definindo o timezone. Mas no fim é só uma questão de como você quer tratar mesmo.